modules/googleplay/src/main/scala/cards/nine/googleplay/processes/CardsProcesses.scala

```diff
@@ -126,7 +127,7 @@ class CardsProcesses[F[_]](
     }
 
     for {
-      list ← cacheService.listPending(numApps)
+      list ← toFree(cacheService.listPending(numApps))
       status ← list.traverse[Free[F, ?], (Package, PackageStatus)] { pack ⇒
         for (status ← resolvePendingPackage(pack)) yield (pack, status)
```

Comment:
Oops, I didn't look to close enough apparently, sorry for the confusion.

`Functor#tupleLeft` might be an alternative.

```scala
resolvePendingPackage(pack).tupleLeft(pack)
```